src/Cards/CurrentCard.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import {
+  Card,
+  Question,
+  Option,
+  SubmitButton,
+  CardWindowBar,
+  ButtonContainer,
+} from "./styled";
+
+import close from "../images/close.svg";
+
+const CurrentCard = ({
+  question,
+  setScore,
+  score,
+  getNextQuestion,
+  isSubmit,
+  setIsSubmit,
+  isCorrect,
+  setIsCorrect,
+  isIncorrect,
+  setIsIncorrect,
+  questionNumber,
+}) => {
+  const [selectedOption, setSelectedOption] = useState("");
+  const [isEmpty, setIsEmpty] = useState(false);
```

Comment:
I was a little confused by what `isEmpty` meant until I looked at when it was being called. A better name would be `hasUserSubmittedAnswer`